fix: provisioner root_path config option

xadips opened the pull request:
# Description

Fixed provisioner `root_path` configuration variable to change hardcoded paths.

All other kitchen drivers default to `/tmp/kitchen` it's harder to adapt cookbooks to allow for testing using `kitchen-dokken` when a default path changes to `/opt/kitchen` (escpecially when testing same cookbooks with both drivers), any idea/reason why it was changed to `/opt/kitchen` to begin with, maybe it should match other drivers for consistency?

## Issues Resolved

Currently, setting `root_path` to anything else other than default value of `/tmp/kitchen` will fail as there's multiple hardcoded paths to `/opt/kitchen`

## Check List

- [ ] New functionality includes tests
- [ ] All tests pass
- [ ] Commit message includes a [Conventional Commit Message](https://www.conventionalcommits.org/en/v1.0.0)

damacus commented on lib/kitchen/driver/dokken.rb:
```diff
@@ -287,7 +287,8 @@ def calc_volumes_binds
 
         binds_ret = []
-        binds_ret << "#{dokken_kitchen_sandbox}:/opt/kitchen" unless dokken_kitchen_sandbox.nil? || remote_docker_host? || running_inside_docker?
+        root_path = instance.provisioner[:root_path]
```
The default path should be '/opt/kitchen'.

xadips commented:
Switched to fetch even though, that method only exists for hash objects :facepalm:  should be fixed now
synced my fork and rebased on top of it

Please re-run tests again
